prime-router/metadata/hl7_mapping/ORU_R01/base/observation-result.yml

```diff
@@ -0,0 +1,125 @@
+name: ORU-R01-observation-result
+constants:
+  hl7OBXField: /PATIENT_RESULT/ORDER_OBSERVATION(%{orderIndex})/OBSERVATION(%{resultIndex})/OBX
+elements:
+  - name: result-set-ID
+    value: [ '%resultIndex + 1' ]
+    hl7Spec: [ '%{hl7OBXField}-1' ]
+
+  - name: result-value-type-st
+    value: [ '''ST''' ]
+    hl7Spec: [ '%{hl7OBXField}-2' ]
+
+  - name: result-value-type-cwe
+    condition: '%resource.value.exists()'
+    value: [ '''CWE''' ]
+    hl7Spec: [ '%{hl7OBXField}-2' ]
+
+  - name: observation-identifier-code
+    value: [ '%resource.code.coding.code' ]
+    hl7Spec: [ '%{hl7OBXField}-3-1' ]
+
+  - name: observation-identifier-text
+    value: [ '%resource.code.text' ]
+    hl7Spec: [ '%{hl7OBXField}-3-2' ]
+
+  - name: observation-identifier-coding-system
+    value: [ '%resource.code.coding.version' ]
+    hl7Spec: [ '%{hl7OBXField}-3-3' ]
+
+  # override to handle LN translated to loinc L on HL7->FHIR conversion
+  - name: observation-identifier-coding-system-ln
+    resource: '%resource.code.coding'
+    condition: '%resource.system = "http://loinc.org" and %resource.version = "L"'
```

Comment:
I think Patricia added something for this.  Check out her code.

Reply:
I can't use it here. The mapping is just different enough that it doesn't play nice with it.